site2/docs/client-libraries-java.md

```diff
@@ -212,7 +212,7 @@ The following is an example.
 
 ```java
 producer.sendAsync("my-async-message".getBytes()).thenAccept(msgId -> {
-    System.out.printf("Message with ID %s successfully sent", msgId);
+    System.out.printf("Message with ID %s successfully sent%n", msgId);
```

Comment:
Good catch, maybe we could even use the more common: 
```suggestion
    System.out.println("Message with ID " + msgId + " successfully sent");
```

Reply:
I am all for it. The tutorial is very beginner friendly, but the `printf` is kinda uncommon in tutorials?